dcs/country.py

```diff
@@ -82,13 +82,12 @@ def find_group(self, group_name, search="exact"):
         return None
 
     def find_group_by_id(self, group_id: int) -> Optional[Group]:
-        groups = [self.vehicle_group,
-                  self.ship_group,
-                  self.plane_group,
-                  self.helicopter_group,
-                  self.static_group]
+        groups: List = [self.vehicle_group,
```

Comment:
This is equivalent to `List[Any]`, which you definitely don't want (try `print(group.fsdafdsafg)` with this, the type checker will not complain). You need to do what I said in the comment: `List[Sequence[Group]]`.